package.json

```diff
@@ -66,12 +66,6 @@
     "typescript": "~4.9.5",
     "which": "^5.0.0"
   },
-  "pnpm": {
-    "overrides": {
-      "typescript": "~4.9.5",
-      "node-gyp": "10.0.1"
-    }
-  },
   "engines": {
```

Comment:
This should be kept

Reply:
I find that surprising and you're absolutely right!